DwCASchemaGenerator/src/main/java/org/gbif/DwCAAvroSchemaGenerator.java

```diff
@@ -0,0 +1,48 @@
+package org.gbif;
```

Comment:
This project doesn't follow the name convention for Maven artifacts, a better approach is to generate  Avro schemas form XSD or XML files, there are several libraries for that, take for example: https://github.com/elodina/xml-avro